### PR TITLE
Adding caption, title to Mplussier

### PR DESCRIPTION
**Pull request by mplussier**
any comments?

### CODE REVIEW

**Review by jingjingli2 — COMMENTED** [review]
very nice change

**Comment by roykyr** [comment]
This is so good, I buy it!

**Comment by Bernard-CRC** [comment]
__great__
👍

**Review by roykyr — APPROVED** [review]
All good :+1:

File: _posts/0000-01-16-mplussier.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Yoda, your every day advisor"
```
Comment (jingjingli2) [inline _posts/0000-01-16-mplussier.md:3]: good

Comment (mplussier) [reply _posts/0000-01-16-mplussier.md:3]: ok thanks

File: _posts/0000-01-16-mplussier.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![yoda](https://cloud.githubusercontent.com/assets/16547949/25401233/2c9215d8-29c3-11e7-9e9b-d7cbccefb17a.jpg)
 {: .center}
 
-Caption here
+�Truly wonderful, the mind of a child is.� � Yoda
```
Comment (roykyr) [inline _posts/0000-01-16-mplussier.md]: Of course.

File: _posts/0000-01-16-mplussier.md [hunk 3]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![yoda](https://cloud.githubusercontent.com/assets/16547949/25401233/2c9215d8-29c3-11e7-9e9b-d7cbccefb17a.jpg)
```
Comment (roykyr) [inline _posts/0000-01-16-mplussier.md:6]: Could be taller, but ok.

**Pull request merged**